src/ui/public/vislib/lib/dispatch.js

```diff
@@ -310,5 +311,11 @@ export default function DispatchClass(Private) {
   }
 
 
+  function justifyOpacity(opacity) {
+    const decimalNumber = parseFloat(opacity, 10);
+    return (0 <= decimalNumber  && decimalNumber <= 1) ? decimalNumber : 0.5;
```

Comment:
No strong preference either. I just felt that it might be more reasonable since a `opacity` of `2` in CSS is treated as a `1`, and a negative `opacity` is treated as `0`.

Then again, users may not even know what CSS is.